### DISCUSSION TITLE
RefCursor

### DISCUSSION THREAD

**Discussion by skysegbr** [oracle/python-oracledb#134] [General]
I'm trying to iterate a cursor of type SYS_REFCURSOR in the same way used in the cx_Oracle version but it's not working in the new oracledb version

it's ok:
cursor = conn.cursor()
refcursor = cursor.var(cx_Oracle.CURSOR)

it's not ok:
refcursor = coracledb.var(CURSOR)

The type CURSOR exists, but it does not work.

Any idea?

**Comment by anthony-tuininga** [comment]
If you were using this before:

```python
cursor = conn.cursor()
refcursor = cursor.var(cx_Oracle.CURSOR)
```

then in python-oracledb you would use this:

```python
cursor = conn.cursor()
refcursor = cursor.var(oracledb.CURSOR)
```

The only way a plain `CURSOR` would work is if you *imported* the value from cx_Oracle or oracledb!
